app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/Accordion.kt

```diff
@@ -0,0 +1,145 @@
+package com.hedvig.android.design.system.hedvig
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.dp
+import com.hedvig.android.design.system.hedvig.AccordionDefaults.Size
+import com.hedvig.android.design.system.hedvig.tokens.AccordionTokens
+import com.hedvig.android.design.system.hedvig.tokens.ColorSchemeKeyTokens.TextAccordion
+import com.hedvig.android.design.system.hedvig.tokens.ColorSchemeKeyTokens.TextPrimary
+
+@Composable
+fun AccordionList(items: List<AccordionData>, modifier: Modifier = Modifier, size: Size = Size.Small) {
+  Column(modifier) {
+    for ((index, perilItem) in items.withIndex()) {
+      var isExpanded by rememberSaveable { mutableStateOf(false) }
+      AccordionItem(
+        isExpanded = isExpanded,
+        onClick = {
+          isExpanded = !isExpanded
+        },
+        title = perilItem.title,
+        description = perilItem.description,
+        size = size,
+      )
+      if (index != items.lastIndex) {
+        Spacer(Modifier.height(4.dp))
+      }
+    }
+  }
+}
+
+@Composable
+fun AccordionItem(
+  isExpanded: Boolean,
+  onClick: () -> Unit,
+  title: String,
+  description: String,
+  modifier: Modifier = Modifier,
+  size: Size = Size.Small,
+) {
+  ExpandablePlusCard(
```

Comment:
I wonder if `ExpandablePlusCard` is gonna be used anywhere else now outside of the accordion component.
If not, it should be moved here and be private, and we never refer to it from anywhere else. And we can update all call-sites to use the Accordion component instead. If this is the case, we can always do it at a later PR

Reply:
Yep